packages/e2e-tests/src/specs/editor/floatingMenu.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+    createNewStory,
+    waitForCarousel
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Floating Menu', () => {
+    beforeEach(async () => {
+        await createNewStory();
+    });
+
+    it('should display text floating menu', async () => {
+        // Open a text tab
+        await expect(page).toClick('#library-tab-text');
+
+        // Add a paragraph
+        await expect(page).toClick('button[data-testid="preview-text"] span', {
+            text: /^Paragraph/,
+        });
+
+        // Floating menu should show up
+        await expect(page).toMatchElement('div[data-testid="context-menu-list"]');
```

Comment:
I think we should go here with something more specific (there can be other `context-menu-list` on the page that is not the floating menu).

Something like https://github.com/GoogleForCreators/web-stories-wp/blob/021448ee508994be9aabeb945113a90f83efa228/packages/story-editor/src/karma/fixture/containers/canvas.js#L83 ?

Reply:
I'm adding a specific testid to the floating menu wrapper

Reply:
Ideally, we should locate the elements by something like a label, that way even if we change the underlying code (and e.g. remove ids/testids) the test will still work (and it has other benefits). @barklund WDYT?

Reply:
@merapi  I've removed the testid and  used `role="menu"`